src/Components/Locations.js

```diff
@@ -0,0 +1,61 @@
+// "https://ghibliapi.herokuapp.com/locations/"
+import axios from "axios";
+import { Component } from "react";
+
+class Locations extends Component {
+  state = { places: [], showLoca: false };
+
+  fetchLocations = async () => {
+    try {
+      const res = await axios.get("https://ghibliapi.herokuapp.com/locations/");
+      this.setState({ places: res.data.map((location) => {
+          location = {name: location.name, climate: location.climate, terrain: location.terrain}
```

Comment:
map works by returning inside the callback function. You can't reassign it's value here. Instead return { name: etc.... }

Reply:
This is what really got you.